env/dev/env/config.cljs

```diff
@@ -1,6 +1,6 @@
 (ns env.config)
 
 (def figwheel-urls {:android "ws://localhost:3449/figwheel-ws",
- :ios "ws://192.168.0.9:3449/figwheel-ws",
+ :ios "ws://10.0.1.4:3449/figwheel-ws",
```

Comment:
I think this one is committed by mistake.

I wonder how can we avoid changing this file all the time... maybe we can read from an env variable?

Reply:
I think we can just add this to gitignore?

Reply:
fixed